include/vsg/ui/PointerEvent.h

```diff
@@ -22,11 +22,11 @@ namespace vsg
     enum ButtonMask : uint16_t
     {
         BUTTON_MASK_OFF = 0,
-        BUTTON_MASK_1 = 256,
-        BUTTON_MASK_2 = 512,
-        BUTTON_MASK_3 = 1024,
-        BUTTON_MASK_4 = 2048, /// mouse wheel up
-        BUTTON_MASK_5 = 4096  /// mouse wheel down
+        BUTTON_MASK_1   = 0x0100, /// left mouse button
+        BUTTON_MASK_2   = 0x0200, /// middle mouse button
+        BUTTON_MASK_3   = 0x0400, /// right mouse button 
+        BUTTON_MASK_4   = 0x0800, /// X1 mouse button
```

Comment:
This conflicts with the original meaning, mouse wheel up/down which follows X11 convention of button 4 & 5 being reserved for scroll wheel up/down.  We'd need to be careful before changing this.

Changing 256 to 0x0100 etc.  I we are to change then 1 << x would probably be better.